src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,11 +5,27 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
Don't begin class implementation with an empty line. It's unnecessary and doesn't follow Java coding conventions.